tests/acceptance/features/apiSpaces/setQuota.feature

```diff
@@ -0,0 +1,184 @@
+@api
+Feature: Set quota
+  As a user
+  I want to set quota to different users
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  Scenario Outline: admin or space admin sets personal space quota of user with different role
+    Given the administrator has given "Alice" the role "<role>" using the settings api
+    And the administrator has given "Brian" the role "<userRole>" using the settings api
+    When user "Alice" changes the quota of the "Brian Murphy" space to "100" owned by user "Brian"
+    Then the HTTP status code should be "200"
+    And for user "Brian" the JSON response should contain space called "Brian Murphy" and match
+    """
+    {
+      "type": "object",
+      "required": [
+        "quota"
+      ],
+      "properties": {
+        "quota": {
+          "type": "object",
+          "required": [
+            "total"
+          ],
+          "properties": {
+            "total" : {
+              "type": "number",
+              "enum": [100]
+            }
+          }
+        }
+      }
+    }
+    """
+    Examples:
+      | role        | userRole    |
+      | Admin       | Admin       |
+      | Admin       | Space Admin |
+      | Admin       | User        |
+      | Admin       | Guest       |
+      | Space Admin | Admin       |
+      | Space Admin | Space Admin |
+      | Space Admin | User        |
+      | Space Admin | Guest       |
+
+
+  Scenario Outline: non-admin user tries to set personal space quota of user with different role
+    Given the administrator has given "Alice" the role "<role>" using the settings api
+    And the administrator has given "Brian" the role "<userRole>" using the settings api
+    When user "Alice" changes the quota of the "Brian Murphy" space to "100" owned by user "Brian"
+    Then the HTTP status code should be "401"
+    And for user "Brian" the JSON response should contain space called "Brian Murphy" and match
+    """
+    {
+      "type": "object",
+      "required": [
+        "quota"
+      ],
+      "properties": {
+        "quota": {
+          "type": "object",
+          "required": [
+            "total"
+          ],
+          "properties": {
+            "total" : {
+              "type": "number",
+              "enum": [0]
+            }
+          }
+        }
+      }
+    }
+    """
+    Examples:
+      | role  | userRole    |
+      | User  | Space Admin |
+      | User  | User        |
+      | User  | Guest       |
+      | User  | Admin       |
+      | Guest | Space Admin |
+      | Guest | User        |
+      | Guest | Guest       |
+      | Guest | Admin       |
+
+
+  Scenario Outline: admin or space admin user with different space role set quota of a space
```

Comment:
`Admin` or `Space Admin` can edit any space regardless of their role in that space, isn't it?
IMO, `<spaceRole>` doesn't matter here. But you could add another scenario where a user with `manger`, `viewer`, `editor` roles edits the quota.

CC @amrita-shrestha @SwikritiT @SagarGi

Reply:
Scenario for `non-admin user` with `manager`, `viewer` and `editor` role trying to edit quota is on L140.

Reply:
Ahh, yeah. That should cover the space-role part.
I don't know what to do here since admin or space-admin user should be able to edit the quota of any space (without being a member of that space). What I think is assigning `<spaceRole>` is not necessary here but this might also cover other edge cases as well

Reply:
We could add another scenario where `admin` or `space-admin` user can edit the quota of any spaces.